exp/CMakeLists.txt

```diff
@@ -30,6 +27,6 @@ target_link_libraries(${EXECUTABLE} p2p)
 if (ETHASHCL)
 	target_link_libraries(${EXECUTABLE} ethash-cl)
 	target_link_libraries(${EXECUTABLE} ethash)
-	target_link_libraries(${EXECUTABLE} ${OpenCL_LIBRARIES})
+	target_link_libraries(${EXECUTABLE} ${OpenCL_TARGET})
```

Comment:
What is `OpenCL_TARGET`? Can't find it in our files by grepping. Is it a cmake thing?